src/EmailRecoveryManager.sol

```diff
@@ -103,6 +103,16 @@ abstract contract EmailRecoveryManager is
         return recoveryRequests[account];
     }
 
+    /**
+     * @notice Checks if the recovery is activated for a given account
+     * @param recoveredAccount The address of the recovered account for which the activation status is being checked
+     * @return bool True if the recovery request is activated, false otherwise
+     */     
+    function isActivated(address recoveredAccount) public view override returns (bool) {
+        return guardianConfigs[recoveredAccount].threshold > 0;
```

Comment:
I think this function returns false even after the account installs the module because they can install it without setting any guardians as the argument of `onInstall` function.